workshop/exercise-3/README.md

```diff
@@ -1,79 +1,64 @@
-## Setup: Install Istio and Knative on Your Cluster
+## Deploy our First Application to Knative using the Knative Client (kn)
+A relatively new project, the knative client aims to make interacting with Knative a seamless experience for developers. We'll try deploying an application using `kn`, then we'll deploy the same application using `kubectl` and `.yaml` files for additional control. The application for this lab is a simple node.js with express app which returns the first n numbers of the Fibonacci sequence. To use the app, deploy it, and simply make a GET request to the `/` endpoint with a number as the parameter. 
 
-Knative is currently built on top of both Kubernetes and Istio.
-If you want to learn more about Kubernetes or Istio, you can check out the
-labs [Kube101](https://github.com/IBM/kube101/tree/master/workshop) and
-[Istio101](https://github.com/IBM/istio101/tree/master/workshop).
-When you install Knative on IKS, it will install Istio for you
-automatically.
+# Our First Knative Service
+Knative Serving supports deploying and serving of serverless applications and functions. Those applications and functions will automatically scale up, and then back down to zero. In this exercise, we'll use the Knative Serving component to deploy our first application from a container image hosted on dockerhub. 
 
-### Install Knative
+Knative Services automatically manage the whole lifecycle for your workload. A service represents your app on Knative. Services control the creation of other objects to ensure that your app has a route, a configuration, and a new revision for each update of the service.
```

Comment:
If we don't go into more details about configurations and routes it might be better to not even mention them here. In v0.6 they try hard to push people to only work with Services so I think it would be best to see if we can keep the lab limited to just that resource.